### PR TITLE
Fix result releasing to use our chunks

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

**Comment by norkans7** [comment]
yep

**Comment by nicpottier** [comment]
@ericnewcomer plz review since there are some import/export tweaks due to releasing instead of deleting

File: temba/flows/models.py [hunk 1]
```diff
@@ -763,8 +763,8 @@ def delete_results(self):
         # in chunks of 1000, remove any values or flowsteps associated with these runs
         # we keep Runs around for auditing purposes
         for chunk in chunk_list(run_ids, 1000):
-            Value.objects.filter(run__in=run_ids).delete()
-            FlowStep.objects.filter(run__in=run_ids).delete()
+            Value.objects.filter(run__in=chunk).delete()
+            FlowStep.objects.filter(run__in=chunk).delete()
```
Comment (ericnewcomer) [inline temba/flows/models.py]: doh

File: temba/campaigns/models.py [hunk 2]
```diff
@@ -106,7 +106,7 @@ def import_campaigns(cls, exported_json, org, user, same_site=False):
                 # we want to nuke old single message flows
                 for event in campaign.events.all():
                     if event.flow.flow_type == Flow.MESSAGE:
-                        event.flow.delete()
+                        event.flow.release()
```
Comment (ericnewcomer) [inline temba/campaigns/models.py]: Yup, we never delete flows, so single message flows having that same treatment seems correct.

**Comment by ericnewcomer** [comment]
yerp

**Pull request merged**